moveit_core/collision_distance_field/include/moveit/collision_distance_field/collision_distance_field_types.h

```diff
@@ -406,7 +411,7 @@ class PosedBodySphereDecompositionVector
   {
     if (i >= decomp_vector_.size())
     {
-      ROS_INFO_NAMED("collision_distance_field", "No body decomposition");
```

Comment:
It would be better if this was LOGGER_COLLISION_DISTANCE_FIELD.get_child("gradient_info"). Perhaps you can have a static const logger variable in the struct.

Reply:
The loggers with consts are failing, the changes are made to leave it with ```static rclcpp::Logger LOGGER_COLLISION_DISTANCE_FIELD = rclcpp::get_logger("moveit").get_child("collision_distance_field");``` 

The changes are made here: https://github.com/ros-planning/moveit2/pull/65#discussion_r293258198